test/integrationtests/voight_kampff/tools.py

```diff
@@ -83,8 +83,9 @@ def mycroft_responses(context):
     if len(messages) > 0:
         responses = 'Mycroft responded with:\n'
         for m in messages:
+            print(m, m.data)
```

Comment:
Leftover print statement that I assume was for temporary debugging and shouldn't be logged?

Reply:
Indeed, will remove